### PR TITLE
fix url to annot1 and 2

### PR DESCRIPTION
**Pull request by lgatto**

### CODE REVIEW

**Comment by lgatto** [comment]
Hi @zkamvar - I am not quite sure what is happening here. The changes in the episodes are harmless (urls changes) and don't involve any new packages. I was able to build/render locally, but it fails here. Any idea?

**Comment by lgatto** [comment]
@csoneson - do you have an idea?

**Comment by csoneson** [comment]
> @csoneson - do you have an idea?

Not really - it looks like its having problems with the package manager. In `bioc-rnaseq` it also had to install the Bioc packages "manually", but there it seemed to work: https://github.com/carpentries-incubator/bioc-rnaseq/actions/runs/4285478861/jobs/7463831131#step:7:656. Did you try to restart the action to see if it would magically resolve 🙂?

**Comment by lgatto** [comment]
Yes, I did try to turn it off and on again.

**Comment by lgatto** [comment]
Where do you install them manually?

**Comment by csoneson** [comment]
Hm, sorry, I see the link doesn't point to the right place automatically. In the linked action, under `Setup Package Cache` -> `Fortify local {renv} packages` (the last one) -> line 645, it's "Attempting to install missing packages assuming bioc"

**Comment by lgatto** [comment]
I see the line you refer to in the output, but xhat file am I supposed to modify to install packages? Somewhere in `.github/workflows`?

**Comment by zkamvar** [comment]
> Hi @zkamvar - I am not quite sure what is happening here. The changes in the episodes are harmless (urls changes) and don't involve any new packages. I was able to build/render locally, but it fails here. Any idea?

Hello,

I just got back to my desk after being away for a week. I am looking into this now.

**Comment by zkamvar** [comment]
> I see the line you refer to in the output, but what file am I supposed to modify to install packages? Somewhere in `.github/workflows`?

You should not have to modify the any of the files inside of that folder. I suspect the problem is from the recent update in {renv} (similar to https://github.com/rstudio/renv/issues/1155).

**Comment by zkamvar** [comment]
Update: I have been able to reproduce the observed behavior on my machine with renv version 0.17.0 and the latest version on GitHub. I will open an issue on {renv} and downgrade the version on GitHub actions.

**Comment by zkamvar** [comment]
I have patched the workflow for now pending an update from {renv}. If you run into this issue or something similar on your own computer, try downgrading {renv} by using 

```r
install.packages("https://cran.r-project.org/src/contrib/Archive/renv/renv_0.16.0.tar.gz")
```

File: episodes/30-dplyr.Rmd [hunk 1]
```diff
@@ -938,7 +938,7 @@ by clicking on the link and then moving it to the `data/` folder, or
 you can use the R code below to download it directly to the folder.
 
 ```{r, message=FALSE}
-download.file(url = "https://raw.githubusercontent.com/carpentries-incubator/bioc-intro/main/_episodes_rmd/data/annot1.csv",
+download.file(url = "https://raw.githubusercontent.com/carpentries-incubator/bioc-intro/main/episodes/data/annot1.csv",
```
Comment (zkamvar) [inline episodes/30-dplyr.Rmd:941]: I would suggest to use the URL for the lesson website, but this is entirely your preference. 

```suggestion
download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.csv",
```

**Review by zkamvar — APPROVED** [review]

File: episodes/30-dplyr.Rmd (hunk 1 above)
Comment (lgatto) [reply episodes/30-dplyr.Rmd:941]: I don't think this works:

Comment (lgatto) [reply episodes/30-dplyr.Rmd:941]: While
```
> download.file(url = "https://raw.githubusercontent.com/carpentries-incubator/bioc-intro/main/episodes/data/annot1.csv", destfile = "annot1.csv")
trying URL 'https://raw.githubusercontent.com/carpentries-incubator/bioc-intro/main/episodes/data/annot1.csv'
Content type 'text/plain; charset=utf-8' length 826 bytes
==================================================
downloaded 826 bytes
```

**Comment by lgatto** [comment]
Thank you for your help resolving the `renv` issue!

**Comment by lgatto** [comment]
I am only vaguely familiar with `renv`, and certainly not as it is used in actions here. I'll try to read up - feel free to suggest some good references.

**Comment by zkamvar** [comment]
> I am only vaguely familiar with `renv`, and certainly not as it is used in actions here. I'll try to read up - feel free to suggest some good references.

If you are interested in how {sandpaper} uses {renv} under the hood, I wrote up an article about that: https://carpentries.github.io/sandpaper/articles/building-with-renv.html

That being said, I've tried to design the workbench so that you don't have to worry about what {renv} is doing behind the scenes, so please do not stress too much about that.

File: episodes/30-dplyr.Rmd (hunk 1 above)
Comment (zkamvar) [reply episodes/30-dplyr.Rmd:941]: > I don't think this works:
> 
> ```
> > download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs", destfile = "annot1.csv")
> trying URL 'https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs'
> Error in download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs",  : 
>   cannot open URL 'https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs'
> In addition: Warning messages:
> 1: In download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs",  :
>   downloaded length 0 != reported length 9339
> 2: In download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs",  :
>   cannot open URL 'https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs': HTTP status was '404 Not Found'
> ```

There is a typo in your URL. It should end in "csv":


```r
> download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.csv", destfile = "annot1.csv")                                                          
trying URL 'https://carpentries-incubator.github.io/bioc-intro/data/annot1.csv'
Content type 'text/csv; charset=utf-8' length 826 bytes
==================================================
downloaded 826 bytes
```

**Pull request merged**